### PR TITLE
Feature/pelagos 4004 fix template asset references

### PR DESCRIPTION
**Pull request by fightingtexasaggie**

### CODE REVIEW

File: templates/SideBySide/index.html.twig [hunk 1]
```diff
@@ -32,15 +32,15 @@
...
-    <script type="text/javascript" src="{{ asset('js/geoviz.js') }}"></script>
-    <script type="text/javascript" src="{{ asset('js/mapWizard.js') }}"></script>
-    <script type="text/javascript" src="{{ asset('js/gMap.js') }}"></script>
-    <script type="text/javascript" src="{{ asset('js/common.js') }}"></script>
-    <script type="text/javascript" src="{{ asset('js/fileBrowser.js') }}"></script>
+    <script type="text/javascript" src="{{ asset('build/js/pelagosNoty.js') }}"></script>
+    <script type="text/javascript" src="{{ asset('build/js/datasetReview.js') }}"></script>
+    <script type="text/javascript" src="{{ asset('build/js/diff_match_patch.js') }}"></script>
+    <script type="text/javascript" src="{{ asset('build/js/sidebyside.js') }}"></script>
+    <script type="text/javascript" src="{{ asset('build/js/geoviz.js') }}"></script>
```
Comment (praneethpr) [inline templates/SideBySide/index.html.twig:39]: Geoviz needs to be moved too

Comment (fightingtexasaggie) [reply templates/SideBySide/index.html.twig:39]: yeah, I only matched everything in the templates directory...thanks for the catch!

File: assets/static/js/mapWizard.js [hunk 2]
```diff
@@ -82,7 +82,7 @@ function MapWizard(json)
         $("#"+json.descField).hide();
 
         $(document.body).append('<div id="divMapWizard">');
-        $("#divMapWizard").load('geoviz/wizard_dialog.html');
+        $("#divMapWizard").load('build/html/geoviz/wizard_dialog.html');
```
Comment (mickel1138) [inline assets/static/js/mapWizard.js:85]: This does NOT change, it stays in Public.

File: assets/static/js/mapWizard.js [hunk 3]
```diff
@@ -117,7 +117,7 @@ function MapWizard(json)
             var wizPromise = this;
             //Synchonous load of HTML, then append to DIV
             $.ajax({
-                url: 'geoviz/wizard_map.html',
+                url: 'build/html/geoviz/wizard_map.html',
```
Comment (mickel1138) [inline assets/static/js/mapWizard.js:120]: Same here. Unless the file was moved?

File: webpack.config.js [hunk 4]
```diff
@@ -91,6 +91,13 @@ Encore
         }
     )
 
+    .copyFiles(
```
Comment (mickel1138) [inline webpack.config.js:94]: This is not needed, and should be undone.

Comment (fightingtexasaggie) [reply webpack.config.js:94]: undone.

**Pull request merged**